homeassistant/components/here_travel_time/strings.json

```diff
@@ -85,5 +85,24 @@
         }
       }
     }
+  },
+  "entity": {
+    "sensor": {
+      "duration": {
+        "name": "duration"
```

Comment:
Don't know of it actually makes a difference but could you please uppercase it?

Reply:
Nice catch! Thanks!